controls/V-72277.rb

```diff
@@ -45,8 +25,9 @@
   tag "fix": "Remove any found \".shosts\" files from the system.
 
 # rm /[path]/[to]/[file]/.shosts"
+  tag "fix_id": "F-78631r1_fix"
 
   describe command("find / -name '*.shosts'") do
```

Comment:
This should be marked as dangerous and at least include `-xdev -xautofs`

Reply:
@trevor-vaughan added -xautofs but can't do -xdev because there can be (and will likely be) multiple partitions that these files can live on.